### PR TITLE
Seek reception typo

### PR DESCRIPTION
**Pull request by lewri**
Correcting an error not caught in #2125

### CODE REVIEW

**Comment by TheCycoONE** [comment]
Please rebase on the latest master to pull in my vcpkg changes. Appveyor didn't like building from scratch on the old versions

File: CorsixTH/Lua/humanoid_actions/seek_reception.lua [hunk 1]
```diff
@@ -75,7 +75,7 @@ local function action_seek_reception_start(action, humanoid)
     local dist = humanoid.world:getPathDistance(humanoid.tile_x, humanoid.tile_y, x, y)
     -- TODO: Make 'else' method work for all humanoids
     if class.is(humanoid, Vip) or class.is(humanoid, Inspector) then
-      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions_on_my_way_to
+      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions.on_my_way_to
       :format(best_desk.object_type.name))
```
Comment (TheCycoONE) [inline CorsixTH/Lua/humanoid_actions/seek_reception.lua:79]: Our coding convention mentions that this line should have been indented: https://github.com/CorsixTH/CorsixTH/wiki/Coding-Conventions

**Comment by lewri** [comment]
Bit of a nasty rebase but hopefully that works

**Comment by lewri** [comment]
> Building ffmpeg for Release

This is taking a worryingly long time in appveyor

**Comment by TheCycoONE** [comment]
Mind fixing the indent on that :format since you're touching the line?

**Comment by TheCycoONE** [comment]
2x2 indent lewri...

File: CorsixTH/Lua/humanoid_actions/seek_reception.lua [hunk 2]
```diff
@@ -75,12 +75,12 @@ local function action_seek_reception_start(action, humanoid)
     local dist = humanoid.world:getPathDistance(humanoid.tile_x, humanoid.tile_y, x, y)
     -- TODO: Make 'else' method work for all humanoids
     if class.is(humanoid, Vip) or class.is(humanoid, Inspector) then
-      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions_on_my_way_to
-      :format(best_desk.object_type.name))
+      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions.on_my_way_to
+        :format(best_desk.object_type.name))
```
Comment (lewri) [inline CorsixTH/Lua/humanoid_actions/seek_reception.lua:79]:
```suggestion
          :format(best_desk.object_type.name))
```

File: CorsixTH/Lua/humanoid_actions/seek_reception.lua [hunk 3]
```diff
@@ -75,12 +75,12 @@ local function action_seek_reception_start(action, humanoid)
...
 
     else
       humanoid:setDynamicInfoText(_S.dynamic_info.patient.actions.on_my_way_to
-      :format(best_desk.object_type.name))
+        :format(best_desk.object_type.name))
```
Comment (lewri) [inline CorsixTH/Lua/humanoid_actions/seek_reception.lua:83]:
```suggestion
          :format(best_desk.object_type.name))
```

**Pull request merged**